developer-guides/rest-api/rooms/upload/README.md

```diff
@@ -75,7 +102,8 @@ Date: Sun, 28 Jan 2018 11:08:18 GMT
 Connection: keep-alive
 Transfer-Encoding: chunked
 
-{"success":true}
+{"message":{"_id":"HxX7qtrm6azSaMChG","rid":"GENERAL","ts":"2019-03-24T20:28:02.501Z",
+"msg":"This is a message with a file and I want to test it","file":{"_id":"RwiWv3waLHfXLW3mj","name":"temp.txt","type":"text/plain"},"attachments":[{"ts": "1970-01-01T00:00:00.000Z","title":"temp.txt","title_link":"/file-upload/RwiWv3waLHfXLW3mj/temp.txt","title_link_download":true,"type":"file","description":"Simple text file"}],"u":{"_id":"abJaoNiibGs7gP9fe","username":"knrt10","name": "knrt10"},"_updatedAt": "2019-03-24T20:28:02.558Z"},"success": true}
```

Comment:
@knrt10 Can you please try to break these lines here? Because they are with big horizontal scroll :/